src/Raven.Server/Indexing/VoronIndexOutput.cs

```diff
@@ -3,13 +3,12 @@
 using Lucene.Net.Store;
 using Voron.Impl;
 using Voron;
-using Sparrow;
 
 namespace Raven.Server.Indexing
 {
-    public class VoronIndexOutput : BufferedIndexOutput
+    public unsafe class VoronIndexOutput : BufferedIndexOutput
     {
-        public static readonly Slice DataKey = Slice.From(StorageEnvironment.LabelsContext, "_", ByteStringType.Immutable);
+        public static readonly int MaxFileChunkSize = 128 * 1024 * 1024; // temporarily 128MB, 2GB eventually
```

Comment:
We probably want to make sure that this is small anyway, both to make sure that this is always running, but also to avoid having super big transactions. 
Note that we still need to test a transaction that is bigger than 2GB